temporal/api/cloud/identity/v1/message.proto

```diff
@@ -12,10 +12,12 @@ option csharp_namespace = "Temporalio.Api.Cloud.Identity.V1";
 import "google/protobuf/timestamp.proto";
 
 message AccountAccess {
-    // The role on the account, should be one of [admin, developer, read]
+    // The role on the account, should be one of [admin, developer, read, owner, financeadmin]
+    // owner - gives full access to the account, including users, namespaces, and billing
     // admin - gives full access the account, including users and namespaces
     // developer - gives access to create namespaces on the account
     // read - gives read only access to the account
+    // financeadmin - gives read only access and write access for billing
```

Comment:
question: is this role ready yet?

Reply:
Yup. It's there for all accounts and has some permissions.